src/shared_modules/content_manager/src/action.hpp

```diff
@@ -197,6 +232,7 @@ class Action final
     std::thread m_schedulerThread;
     std::atomic<bool> m_schedulerRunning = false;
     std::atomic<bool> m_actionInProgress;
+    std::atomic<bool> m_shouldStop = false;
```

Comment:
Improve (name): Maybe m_shouldRun is a more accurate name? ShouldStop seems more related to a sigkill / sigterm action

Reply:
Changed